src/components/encounter-list/encounter-list.component.tsx

```diff
@@ -68,6 +76,10 @@ function obsArrayDateComparator(left, right) {
   return formatDateTime(right.obsDatetime) - formatDateTime(left.obsDatetime);
 }
 
+export function getTherapyPlan(uuid: string) {
```

Comment:
@kajambiya I wouldn't add this helper function to this generic component. Imagine if we did this from the past, this component would be bloated. I would move this to the [art-therapy-tab.component.tsx](https://github.com/UCSF-IGHS/openmrs-esm-ohri/pull/872/files#diff-7498cdb3de2173857d6b6eb1958955a38fa219d2f737f582519645291768cf78) file.

Reply:
@kajambiya moving approving this since it is functional. But you need to work on the comment left behind by @samuelmale